diff -Naur time-1.9.orig/src/time.c time-1.9/src/time.c
--- time-1.9.orig/src/time.c	2026-01-13 19:55:16.806662962 +0800
+++ time-1.9/src/time.c	2026-01-13 19:55:23.662030784 +0800
@@ -728,7 +728,7 @@
      RESUSE *resp;
 {
   pid_t pid;			/* Pid of child.  */
-  sighandler interrupt_signal, quit_signal;
+  __sighandler_t interrupt_signal, quit_signal;
   int saved_errno;
 
   resuse_start (resp);
